File: VB/WpfApplication1/WizardCustomization/CustomSelectHierarchicalDataSourceColumnsPage.vb
Imports Microsoft.VisualBasic
Imports DevExpress.Data.Browsing.Design
Imports DevExpress.Data.XtraReports.Wizard
Imports DevExpress.Data.XtraReports.Wizard.Presenters
Imports DevExpress.Data.XtraReports.Wizard.Views
Imports System
' ...

Namespace WpfApplication1.WizardCustomization
	Friend Class CustomSelectHierarchicalDataSourceColumnsPage
		Inherits SelectHierarchicalDataSourceColumnsPage(Of ReportModel)
		Public Sub New(ByVal view As ISelectHierarchicalDataSourceColumnsPageView, ByVal pickManager As PickManagerBase, ByVal cache As IColumnInfoCache)
			MyBase.New(view, pickManager, cache)
		End Sub

		Public Overrides Function GetNextPageType() As Type
			Return GetType(ChooseReportLayoutPage(Of ReportModel))
		End Function
	End Class
End Namespace

File: CS/WpfApplication1/WizardCustomization/CustomSelectHierarchicalDataSourceColumnsPage.cs
using DevExpress.Data.Browsing.Design;
using DevExpress.Data.XtraReports.Wizard;
using DevExpress.Data.XtraReports.Wizard.Presenters;
using DevExpress.Data.XtraReports.Wizard.Views;
using System;
// ...

namespace WpfApplication1.WizardCustomization {
    class CustomSelectHierarchicalDataSourceColumnsPage : SelectHierarchicalDataSourceColumnsPage<ReportModel> {
        public CustomSelectHierarchicalDataSourceColumnsPage(
            ISelectHierarchicalDataSourceColumnsPageView view, 
            PickManagerBase pickManager, 
            IColumnInfoCache cache) 
            : base(view, pickManager, cache) { }

        public override Type GetNextPageType() {
            return typeof(ChooseReportLayoutPage<ReportModel>);
        }
    }
}